### PR TITLE
[Behat][Promotion] Percentage discount scenarios implementation

### PR DESCRIPTION
**Pull request by Zales0123**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets |
| License       | MIT
| Doc PR        |

PR based on https://github.com/Sylius/Sylius/pull/4208 and https://github.com/Sylius/Sylius/pull/4176.
Implementation of percentage discount scenarios.

### CODE REVIEW

File: src/Sylius/Behat/Context/Setup/PromotionContext.php [hunk 1]
```diff
@@ -109,4 +117,14 @@ private function getPriceFromString($price)
...
     }
+
+    /**
+     * @param string $discount
+     *
+     * @return float
+     */
+    private function getPercentageFromString($discount)
+    {
+        return ((int) $discount)/100;
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Context/Setup/PromotionContext.php]: `((int) $discount) / 100`

Comment (Zales0123) [inline src/Sylius/Behat/Context/Setup/PromotionContext.php]: :+1:

File: src/Sylius/Component/Promotion/Factory/ActionFactoryInterface.php [hunk 2]
```diff
@@ -25,4 +25,11 @@
      * @return ActionInterface
      */
     public function createFixedDiscount($amount);
+
+    /**
+     * @param float $discount
+     *
+     * @return ActionInterface
+     */
+    public function createPercentageDiscount($discount);
```
Comment (michalmarcinkowski) [inline src/Sylius/Component/Promotion/Factory/ActionFactoryInterface.php]: `$amount`

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**